x/tokenfactory/keeper/createdenom_test.go

```diff
@@ -158,6 +156,10 @@ func (suite *KeeperTestSuite) TestCreateDenom() {
 }
 
 func (suite *KeeperTestSuite) TestUpdateDenom() {
+	largeAllowList := make([]string, 2001)
```

Comment:
Similar comment as above wrt making this exceed the MaxAllowListSize param

Reply:
done